Frontend/MoptPaymentPayone/Components/Classes/PayoneHelper.php

```diff
@@ -144,15 +144,6 @@ public function isShippingAddressToBeCheckedWithBasketValue($config, $basketValu
             return false;
         }
 
-        $userData = Shopware()->Modules()->Admin()->sGetUserData();
-
-        if ($userData['billingaddress'] && userData['shippingaddress'] ){
-            if ($this->areAddressesEqual($userData['billingaddress'],$userData['shippingaddress'])) {
```

Comment:
This check is now being performed differently in [Frontend/MoptPaymentPayone/Subscribers/AddressCheck.php:199](https://github.com/PAYONE-GmbH/shopware-5/pull/137/commits/f1547d8f1e82ebb5176823295fb621bc1bc7def8#diff-d2d11f531fcf6080a1e2e61ce56e2b25R199)